cmd/acraproxy/acraproxy.go

```diff
@@ -275,31 +194,56 @@ func main() {
 		fmt.Printf("Error: can't check is exists acraserver public key %v, got error - %v\n", serverPublicKey, err)
 		os.Exit(1)
 	}
-	if *acraHost == "" {
-		fmt.Println("Error: you must specify host to acra")
-		flag.Usage()
-		os.Exit(1)
-	}
 
 	if *verbose {
+		log.SetLevel(log.InfoLevel)
```

Comment:
i believe that in next commits we should use own logger interface, that is linked to any logging system we want, instead of using several log providers inside code

Reply:
it was one-time tests and I forgot to drop one log system and use the second
I will drop in this PR soon